fuzz/go.mod

```diff
@@ -0,0 +1,10 @@
+module local.tld/fuzz
```

Comment:
Does the fuzzer need to be its own module? AFAIK, having multiple modules in a repository can cause complications in some cases.

Reply:
Unfortunately, yes due to [go-fuzz not yet being able to handle modules](https://github.com/dvyukov/go-fuzz/issues/195). Creating another module is a workaround as described by mvdan [here](https://github.com/dvyukov/go-fuzz/issues/195#issuecomment-523526736). It will not work without it due to SIV `/v2`. 

The only pain points I've been experiencing is the tendency of the tools to automatically adding the go-fuzz module to Caddy's `go.mod` and adding Caddy v1.0.3 to Caddy v2's `go.sum`. I had to keep an eye on them and constantly revert them.

Reply:
Are we going to have to keep go.mod and go.sum maintained then, you think? Dang, I wish they'd make it work with modules... this might be kind of annoying.

Reply:
I'm not sure. Do you think this is a bug? Should it modify the root go.mod and go.sum with dependencies of the submodule?